.github/workflows/deploy_cms.yaml

```diff
@@ -52,6 +52,7 @@ jobs:
         uses: aws-actions/amazon-ecr-login@062b18b96a7aff071d4dc91bc00c4c1a7945b076 # v2.0.1
 
       - name: Build Push and Tag
+        continue-on-error: true
```

Comment:
I would add a comment to specify why 'continue-on-error' is set to true

Reply:
I added the comment, feel free to edit it if you think it's appropriate